pom.xml

```diff
@@ -174,6 +174,10 @@
                   <key>raml_files</key>
                   <value>${ramlfiles_path}</value>
                 </systemProperty>
+                <systemProperty>
+                  <key>jsonschema2pojo.config.includeHashcodeAndEquals</key>
```

Comment:
How is this getting used? Will it generate hashCode and equals methods and verifying it via test `SearchOrderLinesTest#testGetPoLinesByAcqUnits` to check if the PoLines are same?

Reply:
Yes, you are right. This is at least one place where this is useful and my personal opinion that it might be useful in other cases as well